docs/src/main/paradox/routing-dsl/directives/tls-directives/extractClientCertificate.md

```diff
@@ -0,0 +1,28 @@
+# extractClientCertificate
+
+@@@ div { .group-scala }
+
+## Signature
+
+@@signature [TlsDirectives.scala](/akka-http/src/main/scala/akka/http/scaladsl/server/directives/TlsDirectives.scala) { #extractClientCertificate }
+
+@@@
+
+## Description
+
+This directive extracts the client certificate for the client mTLS connection where the request was made.
+
+If there is no client trusted certificate present (can only happen with `setWantClientAuth(true)`) or reject the request with a @apidoc[TlsClientUnverifiedRejection].
```

Comment:
```suggestion
If there is no client trusted certificate present (can only happen with `setWantClientAuth(true)`) or reject the request with a @apidoc[TlsClientUnverifiedRejection].
```

Having trouble understading this sentence.. what happens if no client trusted certificate present? It's rejected right? Is the "or" ok there?

Reply:
Heh, yeah that is incomprehensible. If there is no trusted cert present the request is rejected, I'll look through all of them and update to something more clear.